### PR TITLE
Move action links under the Actions menu for project#users/package#users

### PR DESCRIPTION
**Pull request by dmarcoux**

### CODE REVIEW

**Comment by dmarcoux** [comment]
[[S] Review Action Links on Project#users](https://trello.com/c/vkBb8wDZ/1405-s-review-action-links-on-projectusers)

File: src/api/app/views/webui/shared/user_or_groups_roles/responsive_ux/_actions.html.haml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+- content_for :actions do
+  %li.nav-item
+    = link_to('#', data: { toggle: 'modal', target: '#add-user-role-modal' }, id: 'add-user', class: 'nav-link') do
+      %i.fas.fa-plus-circle
```
Comment (krauselukas) [inline src/api/app/views/webui/shared/user_or_groups_roles/responsive_ux/_actions.html.haml:4]: @dmarcoux I saw that for other font-awesome icons in the action menu, we used ```fa-lg``` class and added a margin to it ```mr-1```.

Comment (dmarcoux) [reply src/api/app/views/webui/shared/user_or_groups_roles/responsive_ux/_actions.html.haml:4]: I just checked and you're right. We do use `fa-lg` and `mr-2`.

Comment (dmarcoux) [reply src/api/app/views/webui/shared/user_or_groups_roles/responsive_ux/_actions.html.haml]: Please have another look.

**Pull request merged**